stubs/Flask/flask/logging.pyi

```diff
@@ -1,8 +1,13 @@
-from typing import Any
+from typing import IO
 
-def wsgi_errors_stream(): ...
-def has_level_handler(logger: Any): ...
+from logging import Handler, Logger
 
-default_handler: Any
+from .app import Flask
 
-def create_logger(app: Any): ...
+wsgi_errors_stream: IO[str]
```

Comment:
I haven't investigated it, but would it make sense to use `_typeshed.wsgi.ErrorStream` here?

Reply:
Potentially, but within Flask it's passed into `logging.StreamHandler`, which takes `IO[str]`. I don't know enough to say that `StreamHandler` will happily accept that more constrained protocol.

Reply:
I just checked `StreamHandler`; it only uses `write()` (and `flush()` and `name` if they are available), so we should be able to use `ErrorStream` here.

Reply:
One motivation to keep `IO[str]` is that `Flask`'s docstring for this says "if you make your own `StreamHandler`, you may want to use this for the stream"; that to me implies that it needs to be type compatible with `StreamHandler`'s `__init__`.

I have no code that depends on this, though. Thanks to people committing their virtualenvs, I can't really search for uses either.

Reply:
`IO` annotations are mostly legacy. Actually I am currently preparing a PR that replaces the `IO` annotation of `StreamHandler.__init__` with a protocol, so this should not be an issue.